index.html

```diff
@@ -14,22 +15,35 @@
         <div class="menu-right">
           <div class="desktop">
             <ul>
-              <li><a href="" class="nav-item">Portfolio</a></li>
-              <li><a href="" class="nav-item">About</a></li>
-              <li><a href="" class="nav-item">Contact</a></li>
+              <li><a href="#portfolio" class="nav-item">Portfolio</a></li>
+              <li><a href="#about-me" class="nav-item">About</a></li>
+              <li><a href="#contact" class="nav-item">Contact</a></li>
               <li>
-                <a href=""><img src="images/icon-email.svg" alt="Email" /></a>
+                <a href="#" id="email-svg"
+                  ><img src="images/icon-email.svg" alt="Email"
+                /></a>
               </li>
             </ul>
           </div>
         </div>
-        <a href="" class="mobile">
-          <img src="images/menu.svg" alt="menu" id="menu-svg" />
+        <a href="#" class="mobile" id="mobile-menu-icon">
+          <!-- <img src="images/menu.svg" alt="menu" id="menu-svg" /> -->
         </a>
       </div>
     </nav>
```

Comment:
- [ ] You did a great job on the functionalities but you need to fix the below style issue and make sure your design matches the [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471&t=WD2ITCrRI85HpDky-0) design as much as possible.
  - Please make sure the logo is not visible when the mobile menu is opened(expended).
  - Please increase the gap between mobile menu options.

### Your design

![image](https://user-images.githubusercontent.com/83803414/215534824-ae4290c3-a8a7-4fec-b15b-9f569f91c22a.png)

### Figma design

![image](https://user-images.githubusercontent.com/83803414/215534896-503f4819-bc2c-4535-aa03-00052813e113.png)